client/src/components/form-inputs/FormSelect.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { FormOptionValue } from '@konomi.ai/c-form';
+import { FormControl } from '@mui/material';
+import { Control, Controller } from 'react-hook-form';
+import { StyledInputLabel, StyledSelect, StyledMenuItem } from './styled';
+
+export interface ISelectProps {
+  control: Control;
+  name: string;
+  label: string;
+  required?: boolean;
+  options: FormOptionValue[];
+  multiple?: boolean;
+}
+
+export const FormSelect = ({
```

Comment:
Component should support passing children as options instead of just rendering a data array